src/app/results/results.component.spec.ts

```diff
@@ -93,4 +93,44 @@ describe('ResultsComponent', () => {
     expect(compiled.querySelector('app-advancedsearch')).toBeTruthy();
   });
 
+  it('should have an app-related-search element', () => {
+    let compiled = fixture.debugElement.nativeElement;
+
+    expect(compiled.querySelector('app-related-search')).toBeTruthy();
+  });
+
+  it('should have an app-advancedsearch element', () => {
+    let compiled = fixture.debugElement.nativeElement;
+
+    expect(compiled.querySelector('app-advancedsearch')).toBeTruthy();
+  });
+
+  it('should have an app-infobox element', () => {
+    let compiled = fixture.debugElement.nativeElement;
+
+    expect(compiled.querySelector('app-infobox')).toBeTruthy();
+  });
+
+  it('should have an app-theme element', () => {
+    let compiled = fixture.debugElement.nativeElement;
+
+    expect(compiled.querySelector('app-theme')).toBeTruthy();
+  });
+
+  it('should have a pagination bar', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    expect(compiled.querySelector('div.pagination-bar'));
+  });
```

Comment:
Remove it. As we are going to implement a separate component for it. We don't need it right now.